json-guides/bulkhead.json

```diff
@@ -374,7 +376,8 @@
             "title": "Adding the @Fallback annotation",
             "description": [  
                 "Using the <code>@Asynchronous</code> and <code>@Bulkhead</code> annotations together places requests for a financial advisor in a waiting queue after the number of requests exceeds the specified maximum number of concurrent requests. If the waiting queue is full, a <code>BulkheadException</code> is thrown. Let's add a fallback service to handle the exception. A fallback service runs when the main service fails. It can provide graceful failure or continued or partial operation of the original service.<br>", 
-                "The <code>@Fallback</code> annotation identifies a class that automatically runs when a <code>BulkheadException</code> occurs. In our scenario, we add the <code>ServiceFallbackHandler.class</code> which implements the <code>FallbackHandler.class</code> to allow scheduling of an appointment. When a customer makes a request for a chat session that cannot be handled because the maximum limit of concurrent requests has been reached and the wait queue is full, the <code>ServiceFallbackHandler.handle</code> method is called. The return type of both methods, <code>ServiceFallbackHandler.handle</code> and <code>BankService.serviceForVFA</code>, must be of type <code>Service</code>. Otherwise a <code>FaultToleranceDefinitionException</code> is thrown."
+                "The <code>@Fallback</code> annotation identifies a class or <a href='https://openliberty.io/guides/circuit-breaker.html#adding-the-fallback-annotation'>a method</a> that automatically runs when a <code>BulkheadException</code> occurs. In our scenario, we add the <code>ServiceFallbackHandler.class</code> which implements the <code>FallbackHandler.class</code> to allow scheduling of an appointment. When a customer makes a request for a chat session that cannot be handled because the maximum limit of concurrent requests has been reached and the wait queue is full, the <code>ServiceFallbackHandler.handle</code> method is called. The return type of both methods, <code>ServiceFallbackHandler.handle</code> and <code>BankService.serviceForVFA</code>, must be of type <code>Service</code>. Otherwise a <code>FaultToleranceDefinitionException</code> is thrown.<br>",
+                "Modify your <code>server.xml</code> file to <a href='#enabling-microprofile-fault-tolerance'>include the fault tolerance feature</a>."
```

Comment:
Again....this seems out of place.   I would move it to the paragraph above.....

The <code>@Fallback</code> annotation identifies a class or <a href='https://openliberty.io/guides/circuit-breaker.html#adding-the-fallback-annotation'>a method</a> that automatically runs when a <code>BulkheadException</code> occurs.  To use the <code>@Fallback</code> annotation you must modify your <code>server.xml</code> file to <a href='#enabling-microprofile-fault-tolerance'>include the fault tolerance feature</a>. In our scenario, we add.......        ....then the rest of the paragraph......